frontend/types/api/block.ts

```diff
@@ -0,0 +1,196 @@
+// Code generated by tygo. DO NOT EDIT.
+/* eslint-disable */
+import type { ApiDataResponse, Hash, Address, ClElValue } from './common'
+
+//////////
+// source: block.go
+
+/**
+ * count indicator per block details tab; each tab is only present if count > 0
+ */
+export interface BlockSummary {
+  transactions: number /* uint64 */;
+  votes: number /* uint64 */;
+  attestations: number /* uint64 */;
+  withdrawals: number /* uint64 */;
+  bls_changes: number /* uint64 */;
+  voluntary_exits: number /* uint64 */;
+  blobs: number /* uint64 */;
+}
+export type InternalGetBlockResponse = ApiDataResponse<BlockSummary>;
+export interface BlockExecutionPayload {
+  block: number /* uint64 */;
+  block_hash: Hash;
+  parent_hash: Hash;
+  priority_fees_recipient: Address;
+  priority_fees: string /* decimal.Decimal */;
+  gas_used: number /* uint64 */;
+  gas_limit: number /* uint64 */;
+  base_fee_per_gas: string /* decimal.Decimal */;
+  base_fees: string /* decimal.Decimal */;
+  transactions: {
+    general: number /* uint64 */;
+    blob: number /* uint64 */;
+  };
+  time: number /* int64 */;
+  extra_data?: string;
+  graffiti: string;
+}
+export interface BlockConsensusLayer {
+  state_root: Hash;
+  signature: Hash;
+  randao_reveal: Hash;
+  attestations: number /* uint64 */;
+  votes: number /* uint64 */;
+  voting_validators: number /* uint64 */;
+  voluntary_exits: number /* uint64 */;
+  attester_slashings: number /* uint64 */;
+  proposer_slashings: number /* uint64 */;
+  deposit: number /* uint64 */;
+  sync_committee: BlockSyncCommittee;
+  eth1_data: BlockEth1Data;
+}
+export interface BlockSyncCommittee {
+  participation: number /* float64 */;
+  bits: boolean[];
+  sync_committee: number /* uint64 */[];
+  signature: Hash;
+}
+export interface BlockEth1Data {
+  block_hash: Hash;
+  deposit_count: number /* uint64 */;
+  deposit_root: Hash;
+}
+export interface BlobInfo {
+  name: number /* uint64 */;
+  tx_count: number /* uint64 */;
+  gas_used: number /* uint64 */;
+  gas_price?: string /* decimal.Decimal */;
+  excess_gas: number /* uint64 */;
+  burned_fees?: string /* decimal.Decimal */;
+}
+export interface BlockOverview {
+  /**
+   * General
+   */
+  block: number /* uint64 */;
```

Comment:
Some properties overlap between the `BlockOverview` and the `BlockExecutionPayload`. 
Wouldn't it be enough to have them just in the  `BlockExecutionPayload` ?